pdc/apps/component/tests.py

```diff
@@ -1175,6 +1175,19 @@ def test_clone_components_bad_include_inactive_value(self):
         self.assertIn('Value [foobar] of include_inactive is not a boolean',
                       response.content)
 
+    def test_release_components_clone(self):
+        args = {"name": "Supplementary", "short": "supp", "version": "1.1",
+                "release_type": "ga"}
+        target_response = self.client.post(reverse('release-list'), args)
+        self.assertEqual(target_response.status_code, status.HTTP_201_CREATED)
+        target_release_id = target_response.data['release_id']
+        response = self.client.post(reverse('releasecomponentclone-list'),
+                                    {'source_release_id': 'release-1.0',
+                                     'target_release_id': target_release_id,
+                                     'component_dist_git_branch': 'new_branch'},
+                                    format='json')
+        self.assertEquals(response.status_code, status.HTTP_201_CREATED)
+
```

Comment:
I think more test cases should be added. For example,the test case for no releasecomponent in source release or there is inactive releasecomponent, etc.